app/javascript/components/item-details.vue

```diff
@@ -2,18 +2,26 @@
 import { computed, ref } from 'vue';
 import { useNotification } from '@kyvg/vue3-notification';
 import type { Item } from '../api/items';
+import type { Review } from '../api/reviews';
 import purchasesApi from '../api/purchases';
+import reviewApi from '../api/reviews';
+import ItemReviewList from './item-review-list.vue';
+import ItemReviewOption from './item-review-option.vue';
+import ItemReviewForm from './item-review-form.vue';
 
 type Props = {
   item: Item
+  reviews: Review[]
 };
-const props = defineProps<Props>();
 
+const props = defineProps<Props>();
 const currencyFormatter = new Intl.NumberFormat('es-CL', { style: 'currency', currency: 'CLP' });
 const formattedPrice = computed(() => currencyFormatter.format(props.item.price));
-
-const loading = ref(false);
+const writeReviewOption = ref(false);
```

Comment:
Qué representa esta variable? Por el nombre me da la imporesión que representa siesque está habilitada la opción de escribir una review? Pero se me hace raro que parta en false, entonces quizás no la entendí

Reply:
Ahora pensándolo, creo que tiene un mal nombre, porque se encarga de mostrar o no el form para ingresar una review. Cuando está en false se muestra esto:
![image](https://github.com/platanus/the-store1/assets/69608419/929c3ee1-76da-43ac-a49e-7897ccc99c36)

Y cuando está en true se muestra el form:
![image](https://github.com/platanus/the-store1/assets/69608419/2522af10-8ffa-43c5-be41-cb6421fa681f)

Son componentes distintos y controlo que se muestren con un v-if 
![image](https://github.com/platanus/the-store1/assets/69608419/8de35049-cbd0-4911-8a1b-7d08e91adf2d)

Reply:
quizás puede llamarse `showingWriteReviewInput`